src/payment/PaymentPage.jsx

```diff
@@ -133,7 +137,13 @@ class PaymentPage extends React.Component {
             <PlaceOrderButton />
           ) : (
             <React.Fragment>
-              <PaymentMethodSelect loading={loading} isBasketProcessing={isBasketProcessing} />
+              <PaymentMethodSelect
+                submitPayment={this.props.submitPayment}
+                submitPaymentSuccess={this.props.submitPaymentSuccess}
```

Comment:
If the ApplePayButtonContainer was not part of the ApplePay "module" we could logically just connect that to these actions and skip passing props down through multiple components.

Reply:
Truth